webapp/commonComponents/form/input.js

```diff
@@ -26,6 +26,7 @@ export const Input = React.forwardRef((props, ref) => {
   return (
     <ValidationTooltip
       validation={validation}
+      showKeys={false}
```

Comment:
input is generic. should showKeys be passed from the caller?

Reply:
In fact showKeys is not needed in this case, as there's no 'field' attribute in the validation object, I removed showKeys in <validationTooltip />